acquia_lift.links.action.yml

```diff
@@ -0,0 +1,5 @@
+entity.acquia_lift_slot.add_form:
+  route_name: entity.acquia_lift_slot.add_form
```

Comment:
I would try "entity.acquia_lift.slot.add_form" and "entity.acquia_lift.slot.add_form" first, which is what I have been doing with past Symfony projects. Did Drupal 8 enforced a new convention?